nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -148,6 +148,24 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 - (Neo)Vim can not be configured with `configure.pathogen` anymore to reduce maintainance burden.
 Use `configure.packages` instead.
 
+- The WordPress module now stores themes and plugins in a directory name without appending the current version. This avoids loosing settings when upgrading themes and plugins as some of them use the directory name to store the settings in the database. This needs manual intervention in case of using such a broken plugin or theme which is rare.
+  ```bash
+  nix-shell -p wp-cli
+  cat /etc/httpd/httpd.conf  | grep "\-wordpress-"
+  cd /nix/store/xxx-wordpress-xxx/share/wordpress
+  sudo -u wordpress wp maintenance-mode activate
+  sudo -u wordpress wp maintenance-mode status
+  sudo -u wordpress wp db export
+  sudo -u wordpress wp db search name-version
+  sudo -u wordpress wp search-replace --dry-run --verbose --precise name-version name
+  # repeat without --dry-run if satisfied with results
+  sudo -u wordpress wp maintenance-mode deactivate
+  ```
+  (https://developer.wordpress.org/cli/commands/). Afterwards verify that the website is working fine and no settings got lost. If case of loss you can recover using the exported sql dump.
```

Comment:
```suggestion
  (https://developer.wordpress.org/cli/commands/). Afterwards verify that the website is working fine and no settings got lost. In case of loss you can recover using the exported sql dump.
```

Add program listing to import.